dotcom-rendering/src/components/Badge.tsx

```diff
@@ -37,12 +37,26 @@ const badgeLink = css`
 type Props = {
 	imageSrc: string;
 	href: string;
+	ophanComponentLink?: string;
+	ophanComponentName?: string;
 	isInLabsSection?: boolean;
 };
 
-export const Badge = ({ imageSrc, href, isInLabsSection = false }: Props) => {
+export const Badge = ({
+	imageSrc,
+	href,
+	ophanComponentLink,
+	ophanComponentName,
+	isInLabsSection = false,
+}: Props) => {
 	return (
-		<a href={href} css={badgeLink} role="button">
+		<a
+			href={href}
+			data-link-name={ophanComponentLink}
+			data-component={ophanComponentName}
+			css={badgeLink}
+			role="button"
```

Comment:
Not your change (introduced [here](https://github.com/guardian/dotcom-rendering/pull/925)), but I can't see why we're putting `role="button"` on a link. It might be a nice accessibility bonus to remove this line, if you fancy testing in VoiceOver, and have the time 🙂

Reply:
I haven't tried testing in VoiceOver before but I am interested to learn about it so I will merge this PR and that could be a change for another time. Thanks Simon for pointing it out 🙂